src/Facets/DexManagerFacet.sol

```diff
@@ -47,7 +47,12 @@ contract DexManagerFacet {
             if (dex == address(this)) {
                 revert CannotAuthoriseSelf();
             }
-            if (LibAllowList.contractIsAllowed(dex)) continue;
+            if (LibAllowList.contractIsAllowed(dex)) {
```

Comment:
I would propose to add a small test case that batchAdds some addresses (of which one is already whitelisted) and makes sure that all addresses are whitelisted after the (successful) execution.